onset_maproom/Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM centos:7.9.2009
+
+# yum packages
+# httpd-devel and gcc are to support building mod_wsgi from source.
+RUN yum install -y httpd httpd-devel gcc
+
+RUN rm -r /etc/httpd/conf.d /etc/httpd/conf.modules.d
+
+
+# miniconda
+RUN curl -L \
+    https://repo.anaconda.com/miniconda/Miniconda3-py39_4.9.2-Linux-x86_64.sh \
+    -o /miniconda-installer.sh && \
+    bash /miniconda-installer.sh -b -p /conda && \
+    eval "$('/conda/bin/conda' 'shell.bash' 'hook' 2> /dev/null)" && \
+    conda config --set auto_update_conda False && \
+    conda install conda==4.10.1
+
+# conda environment
+COPY environment_linux.yml /build/environment.yml
+RUN eval "$('/conda/bin/conda' 'shell.bash' 'hook' 2> /dev/null)" && \
+    conda env create -f /build/environment.yml -n app
+
+# mod_wsgi: use pip to compile mod_wsgi from source for the particular versions
+# of apache and python that we're using.
+RUN eval "$('/conda/bin/conda' 'shell.bash' 'hook' 2> /dev/null)" && \
+    conda activate app && \
+    pip install mod_wsgi==4.7.1
+
+RUN yum -y install mesa-libGL
```

Comment:
This had to be installed manually and I couldn't figure out which conda package would add it. This might be non-ideal (since it's versioned separately) or it might not matter.

Reply:
This is how we do it in fbfmaproom. https://bitbucket.org/iridl/x24-vector/src/ca47ae24bdbcdb2dbc5d6ba0a9e2f8e54a595936/fbfmaproom/Dockerfile#lines-5:7

Reply:
Yeah, no reason to not bundle that with the first yum command